views/base.dt

```diff
@@ -49,4 +39,16 @@ body(ng-app="DlangTourApp", class="ng-cloak")
 						img(src="/static/img/GitHub-Mark-Light-32px.png")
 
 	block content
+
+	block js
```

Comment:
you have to move that to the bottom. Otherwise `tour.dt` will inject it's JS into the element and `tour-controller.js` will be loaded after `angular.js` ...